migrations/20231005230721_dynamic-fields.sql

```diff
@@ -0,0 +1,109 @@
+CREATE TABLE games (
+	id int PRIMARY KEY, -- Only used in db
+	name varchar(64),
+  CONSTRAINT unique_game_name UNIQUE (name)
+);
+INSERT INTO games(id, name) VALUES (1, 'minecraft-java');
+INSERT INTO games(id, name) VALUES (2, 'minecraft-bedrock');
+
+ALTER TABLE mods ADD COLUMN game_id integer REFERENCES games ON UPDATE CASCADE NOT NULL DEFAULT 1; -- all past ones are minecraft-java
+ALTER TABLE loaders ADD COLUMN game_id integer REFERENCES games ON UPDATE CASCADE NOT NULL DEFAULT 1; -- all past ones are minecraft-java
```

Comment:
I didn't see that, but there is some relevance to it here given that we are doing a lot of loader specific stuff (like loader field enumes enums being game-specific but not loader specific). I included the bare minimum implementation for now, I figure in a future PR where we 'fully' implement games this could be done.